### PR TITLE
Cap

### PR DESCRIPTION
**Pull request by dfaulken**

### CODE REVIEW

**Comment by dfaulken** [comment]
TODO before merge:

```ruby
set :branch, 'master'
```

or remove

File: config/database.yml.example [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: ftforms_test
+
+production:
+ <<: *default
+ adapter: postgresql
+ database: db/production.sqlite3
```
Comment (werebus) [inline config/database.yml.example]: We're not using them, but these settings conflict.

File: config/deploy.rb [hunk 2]
```diff
@@ -1,49 +1,19 @@
...
-
-end
+set :application, 'ft-forms'
+set :repo_url, 'git@github.com:umts/ft-forms.git'
+set :branch, 'cap'
+set :deploy_to, "/srv/#{fetch :application}"
+set :log_level, :info
+set :scm, :git
+set :keep_releases, 5
+set :use_sudo, true
```
Comment (werebus) [inline config/deploy.rb]: Since this didn't fix the directory completion problem, does deploy work without it?

**Comment by dfaulken** [comment]
Okayed by @werebus

**Pull request merged**